sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -186,9 +82,34 @@ public class AADOAuth2LoginConfigSample extends AADWebSecurityConfigurerAdapter
 }
 ```
 
-### [Web APP] Configure scopes of multiple resources
-By default, `azure-spring-boot-starter-active-directory` configures scopes of `openid`, `profile` and `https://graph.microsoft.com/user.read` to implement OpenID Connect protocol and access of membership information of logging in users.
+### Web application with function of visiting resource servers
+
+#### System diagram
+![Web Application Access Resources](resource/add-based-web-application-access-resources.png)
+
+1. Login with credentials, the scope includes default scopes and all configured scopes. 
+2. Auto-acquire the access token of other clients based on the root refresh token.
+3. Use each client's access token to request restricted resource.
+4. Return secured data.
+
+#### Include the package
+To use this starter in a web application, please add following packages:
```

Comment:
Change all to `To use this starter in this scenario, please add the following dependencies in you pom.xml`